superset/db_engine_specs/base.py

```diff
@@ -1433,8 +1433,9 @@ def select_star(  # pylint: disable=too-many-arguments,too-many-locals
         if show_cols:
             fields = cls._get_fields(cols)
         quote = engine.dialect.identifier_preparer.quote
+        quote_schema = engine.dialect.identifier_preparer.quote_schema
```

Comment:
Daydreaming: How I wish Python had this syntax like JS/TS does..
```ts
{ quote, quote_schema } = engine.dialect.identifier_preparer
```

Reply:
Wherever a schema is the input argument, yes. For table names and the like, `quote` is fine. As far as I could tell this is the only place where a schema name is formatted